brooklyn-server/server-cli/src/main/java/org/apache/brooklyn/cli/Main.java

```diff
@@ -474,17 +472,22 @@ public Void apply(CatalogInitialization catInit) {
             }
             
             BrooklynServerDetails server = launcher.getServerDetails();
-            ManagementContext ctx = server.getManagementContext();
+            ManagementContext mgmt = server.getManagementContext();
             
             if (verbose) {
                 Entities.dumpInfo(launcher.getApplications());
             }
             
             if (!exitAndLeaveAppsRunningAfterStarting) {
-                waitAfterLaunch(ctx, shutdownHandler);
+                waitAfterLaunch(mgmt, shutdownHandler);
             }
 
-            // will call mgmt.terminate() in BrooklynShutdownHookJob
+            // BrooklynShutdownHookJob will invoke terminate() to do mgmt.terminate() and BrooklynWebServer.stop();
+            // and System.exit is invoked immediately after ...
+            // but seems better to do it explicitly here. 
+            // ('launcher' is local to us so the caller *cannot* do it, and no harm in doing it twice.) 
+            launcher.terminate();
```

Comment:
thanks!  yes there is stuff in `BrooklynShutdownHookJob` we need to tab into.  i still think we should trigger it manually here, because we can rather than rely on the shutdown hook, but what i've done is definitely NOT good enough.